frr-files/install.sh

```diff
@@ -3,17 +3,18 @@
 set -x
 
 BASE=/frr-files
-NAME=$(ipmitool user list | awk '/OSISM/ {print substr($2,6)}')
-IP=${NAME:5}
+SYSTEM_HOSTNAME="$(ipmitool dcmi get_mc_id_string|awk '/^.*:/{hostname=$0;gsub(/^(..*): /, "", hostname);printf("%s", hostname); exit}')"
+BMC_IP="$(ipmitool lan print |awk '/^IP Address[ ]*:/{hostname=$0;gsub(/^(..*): /, "", hostname);printf("%s", hostname); exit}')"
+IP_SUFFIX="$(echo "$BMC_IP"|grep -P -oh '\d+$')"
 
-echo Name is: $NAME, IP suffix is: $IP
+echo "Name is: $SYSTEM_HOSTNAME, IP suffix is: $IP_SUFFIX, BMC IP is: $BMC_IP"
 
-echo $NAME > /etc/hostname
-echo "\n - \S - IP - 10.10.21.$IP" > /etc/issue
+echo $SYSTEM_HOSTNAME > /etc/hostname
+echo "\n - \S - IP - 10.10.21.$IP_SUFFIX" > /etc/issue
 
 rm -f /etc/netplan/00-installer-config.yaml
-sed -e "s/IP/$IP/" $BASE/01-osism.yaml > /etc/netplan/01-osism.yaml
-sed -e "s/IP/$IP/" -e "s/HOST/$NAME/" $BASE/frr.conf > /etc/frr/frr.conf
+sed -e "s/IP_SUFFIX/$IP_SUFFIX/" $BASE/01-osism.yaml > /etc/netplan/01-osism.yaml
```

Comment:
We use 01-osism.yaml here on all nodes. If we want to change this we have to change osism.commons.network first.

Reply:
Reverted the renaming, we can consider to that that later.